src/App.jsx

```diff
@@ -127,10 +126,13 @@ function App() {
           </Portal>
           {showAccountSwitcher() && <AccountSwitcher />}
           <Routes>
-            <Route element={<Endpoint />} path="/endpoint" />
-            <Route path="/" element={<MarketList />} />
-            <Route path="/trade" element={<Trade />} />
-            <Route path="/reports" element={<Reports />} />
+            <Route path="/endpoint" component={Endpoint} />
+            <Route path="/" component={RouteGuard}>
+              <Route path="/home" component={MarketList} />
+              <Route path="/trade" component={Trade} />
+              <Route path="/reports" component={Reports} />
+            </Route>
+            <Route path="*" component={MarketList} />
```

Comment:
can we create a route config for this?

Reply:
Do we need a config file for this? the app has only 4 routes and I think we wont be adding any more sections?

Reply:
if we dont have many routes, we can leave as it is.. \